packages/context/src/context.ts

```diff
@@ -116,18 +117,32 @@ export class Context extends EventEmitter {
    * upon `bind` and `unbind` events
    */
   private setupEventHandlers() {
+    // tslint:disable-next-line:no-any
+    const notificationErrorHandler = (...args: any[]) => {
+      // Catch error to avoid lint violations
```

Comment:
I am confused about this comment. Which linting rule would be violated without the debug statement?

Reply:
That was a left-over from refactoring. Removed.